### DISCUSSION TITLE
A lot of open-solved topics

### DISCUSSION THREAD

**Discussion by rlalik** [community/community#126443] [Discussions]
### Select Topic Area

Product Feedback

### Body

The discussion boards are flooded with answered topics which are still open. Most likely OPs just are forgetting that they should close the topic if it is fixed for them.

Perhaps there could be some message appearing when OP accepts an answer to remind him about closing? Like a footer in the the accepted answer saying: `Oh, it seems that you have solved your issues. Please consider closing this topic.`.
Or a button to request closing to the OP that he can get notification.
Or closing automatically after a couple of days of inactivity in answered threads? Just an idea.

Cleaning up the open topics from answered help keep tracking of what still should bring others attention.

**Comment by LiteBrite82** [comment]
Hi @rlalik 
Moderators actively curate the discussions boards. In addition, we recently introduced [Stale-bot](https://github.com/orgs/community/discussions/70478#discussion-5750421) into more categories. We should start seeing a reduction in open posts with answers. 
Also, sometimes we intentionally leave answered posts open; depending on the complexity of the discussion.
